CMakeLists.txt

```diff
@@ -68,11 +68,20 @@ endif()
 find_package(VulkanHeaders REQUIRED CONFIG)
 
 # Find and create the Vulkan::Vulkan library
-find_library(Vulkan_LIBRARY NAMES vulkan vulkan-1)
-add_library(Vulkan::Vulkan UNKNOWN IMPORTED)
-set_target_properties(Vulkan::Vulkan PROPERTIES
-    IMPORTED_LOCATION ${Vulkan_LIBRARY})
+add_library(Vulkan::Vulkan SHARED IMPORTED)
 target_link_libraries(Vulkan::Vulkan INTERFACE Vulkan::Headers)
+if (WIN32)
+    find_library(Vulkan_LIBRARY_DLL NAMES vulkan-1 PATH_SUFFIXES bin)
+    find_library(Vulkan_LIBRARY_IMPLIB NAMES vulkan-1)
+    set_target_properties(Vulkan::Vulkan PROPERTIES
+        IMPORTED_LOCATION ${Vulkan_LIBRARY_DLL})
+    set_target_properties(Vulkan::Vulkan PROPERTIES
+        IMPORTED_IMPLIB ${Vulkan_LIBRARY_IMPLIB})
```

Comment:
Part of me is considering adding find_package support to the loader because of this...

Reply:
I am also VERY much in favor of that now as well.

Reply:
https://github.com/KhronosGroup/Vulkan-Loader/pull/1293

Try using my branch:

```
find_package(VulkanLoader)

# Now you can use Vulkan::Vulkan
```